### ISSUE TITLE
Problem with react-native setup

### ISSUE THREAD

**Issue by paluh** [uniphil/nix-for-devs#1]
Could you confirm that your shell.nix works for react-native (I've checked against fresh nix-unstable checkout and using stable channel) because I'm getting:

```
Error: Could not find or load main class com.android.sdklib.tool.SdkManagerCli
```

from these steps:

```
cd .android/sdk
          $PWD/bin/sdkmanager --update
          echo "=> installing platform stuff (you'll need to accept a license in a second)..."
          $PWD/bin/sdkmanager "platforms;android-23" "build-tools;23.0.1" "add-ons;addon-google_apis-google-23"
```

What is worst I'm just starting with android and I haven't touched java for few years. Do you have any suggestions?

**Comment by paluh** [comment]
Nevermind - it was my fault... Project directory name was automaticyally generated by me and had current time info with `:` in it ;-)

**Comment by uniphil** [comment]
Oh wow did it work? It's been a while since I last started a react-native project with these steps, and that world moves fast haha.

Is there anything to tweak that could make it more clear? Maybe adding a note about paths with `:`s in them?

**Comment by paluh** [comment]
Hi @uniphil,

I had a lot of problems down the road, so I've decided to move on and try to build upon existing expression from unstable nixpkg. I've added licensing confirmation to it and I can compile simple projects by changing some versions numbers in some gradle.build scripts. This strategy fails though for things like `gl-react-native` (it crashes with `NullPointerException`).
As your repository is really valuable source of information for developers which are using nix (and nix newbies like me) I think it could be useful to update react-native section and provide some use cases to it.
I can share my experience here in this thread and finally provide PR if you are interested.
